.github/workflows/initiaterelease.yml

```diff
@@ -11,18 +14,23 @@ jobs:
       id-token: write
       contents: write
     env: 
-      IAM_INSTANCE_PROFILE_ARN: ${{secrets.IAM_INSTANCE_PROFILE_ARN}} 
+      IAM_INSTANCE_PROFILE_ARN: ${{ secrets.IAM_INSTANCE_PROFILE_ARN }} 
+      GH_TOKEN: ${{ github.token }}
```

Comment:
Was this line always necessary? Or why is it added as part of this pull request but was not there before?

Reply:
It's necessary to run the gh pr create command, as determined from testing with my own fork. It wasn't able to create a pull request without using the github token of the bot. Here's a link to the failed run: https://github.com/hozkaya2000/amazon-ecs-ami/actions/runs/8012615864/job/21888228787

Under the failed step, we can see that GH_TOKEN should be set.